fastlane/Fastfile

```diff
@@ -469,7 +469,7 @@ ENV["validate_translations"]="lintVanillaRelease"
     {library: "Stories Creator", strings_path: "./libs/stories-android/stories/src/main/res/values/strings.xml", exclusions: []}
   ]
 
-  BINARY_IMPORTED_LIBRARIES = [
+  binary_imported_libraries = [
```

Comment:
Nice catch!

No big deal, but I would suggest keeping this as all uppercase, because that's how [constants are defined in Ruby](https://www.rubyguides.com/2017/07/ruby-constants/).

Reply:
Ahhh you're right 🤦‍♂️ overlooked that detail! I'll get this change in with another PR

Reply:
Yeah no worries. I kept confusing them till it was pointed out to me. It's a curious design choice, imho. Also, it's a minor optimization in the context of our `Fastfile`. No stress 😄